binding/web/README.md

```diff
@@ -91,21 +91,37 @@ npx pvbase64 -h
 
 ### Init options
 
-Rhino saves and caches your model file in IndexedDB to be used by Web Assembly. Use a different `customWritePath`
-variable to hold multiple model values and set the `forceWrite` value to true to force re-save the model file.
-Set `processErrorCallback` to handle errors if an error occurs while transcribing.
+Rhino saves and caches your model (`.pv`) and context (`.rhn`) files in the IndexedDB to be used by Web Assembly.
+Use a different `customWritePath` variable to hold multiple model values and set the `forceWrite` value to true to force a re-save of the model file.
 If the model file (`.pv`) changes, `version` should be incremented to force the cached model to be updated.
 
 ```typescript
-// these are default
+const rhinoContext = {
+  publicPath: ${CONTEXT_RELATIVE_PATH}, // or base64: ${CONTEXT_BASE64_STRING},
+  customWritePath: 'custom_context'; // Optional
+  forceWrite: true; // Optional
+  version: '1.0'; // Optional
+}
+
+const rhinoModel = {
+  publicPath: ${MODEL_RELATIVE_PATH}, // or base64: ${MODEL_BASE64_STRING},
+  customWritePath: 'custom_model'; // Optional
+  forceWrite: true; // Optional
+  version: '1.0'; // Optional
+}
+```
+
+Additional engine options are provided as the `options` parameter.
+Set `processErrorCallback` to handle errors if an error occurs while processing audio.
+Use `endpointDurationSec` and `requireEndpoint` to control endpoint parameters.
+An endpoint is a chunk of silence at the end of an utterance that marks the end of spoken command.
+
+```typescript
+// these are the default values
 const options = {
-  sensitivity: 0.5,
-  endpointDurationSec: 1.0,
-  requireEndpoint: true,
-  processErrorCallback: (error) => {},
-  customWritePath: "rhino_model",
-  forceWrite: false,
-  version: 1,
+  endpointDurationSec: 1.0, // Optional
+  requireEndpoint: true, // Optional
+  processErrorCallback: (error) => {}, // Optional
```

Comment:
instead of writings // Optional per options, in the top comment we can: // Optional, these are the default values